### ISSUE TITLE
Serverside removed recipe can be crafted clientside without using up resources

### ISSUE THREAD

**Issue by Psy-Virus** [modmuss50/RecipeManipulator#4]
When you put a recipe into the `removal.json` only on the server ans not on the client, the client can still craft an item by putting the resources into the crafting table and shift-leftclick the result into the inventory. If the client then opens the inventory and drops the item and gathers it again, the item stays in the inventory.

The best idea to prevent this is probably to sync the removed and added recipes automatically to the client.

EDIT: For example when you remove enderstorage items and blocks by putting
```
{
  "recipesToRemove": [
  "block@enderstorage:ender_storage",
  "item@enderstorage:ender_pouch"
  ]
}
```
into the `removal.json`.

:-)

**Comment by modmuss50** [comment]
Yeah, right now you really should have the same recipes on both the client and the server. If I get a chance I can look into adding it.
